pkg/download/retry_test.go

```diff
@@ -92,7 +92,7 @@ func TestRetriesWith_SwitchDownloaderOn404(t *testing.T) {
 	require.Nil(t, err, "should eventually succeed")
 	require.NotNil(t, resp, "response body exists")
 	require.Equal(t, d404.timesCalled, 1)
-	require.Equal(t, d200.timesCalled, 4)
+	require.Equal(t, d200.timesCalled, 2)
 }
 
 func TestRetriesWith_SwitchDownloaderThenFailWithCorretErrorMessage(t *testing.T) {
```

Comment:
This test is failing on line 111 because Download returned the generic error message rather than the MsiDownload403ErrorString, is this behavior correct? If so then that check should be fixed, the same issue causes the download_test.go/TestDowload_msiDownloaderErrorMessage to fail as well

Reply:
Fixed